README.md

```diff
@@ -57,6 +57,26 @@ Example
 Looking for example to use `react-flexbox-grid`? Head over to [react-flexbox-grid-example](https://github.com/roylee0704/react-flexbox-grid-example).
 
 
+Advanced composition
+-------
+Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you have a `MyFormInput` component that should also act as both a `Row` and a `Col`.
+
+```js
+import {getRowClassNames, getColClassNames} from 'react-flexbox-grid'
+
+export default function MyFormInput(props) {
+  return (
+    <form className={getRowclassNames(props)}>
+      <input className={getColClassNames(props)} />
```

Comment:
Hm, yes good point.  I'm actually feeding it into the `classnames` lib which seems smart enough to be do the right thing.   I'll play with it a bit to find the best approach, then make a PR.